src/components/panel/panel.js

```diff
@@ -35,12 +35,12 @@ angular
  *   var panelRef;
  *
  *   function showPanel($event) {
- *     var panelPosition = $mdPanelPosition
```

Comment:
It's right now.

Basically, I was saying that `$mdPanelPosition` and $mdPanelAnimation` are not services, so they wouldn't have been injected. So without the methods through `$mdPanel`, there's no way anyone could access the objects.